### PR TITLE
Impact discover

### PR DESCRIPTION
**Pull request by jgalobart**
### Files
- HTML: /Resources/templates/responsive/impact_discover/index.php
- JS: /public/assets/js/impact_discover/impact_discover.js
- SASS: /impact-discover/public/assets/sass/layouts/_impact_discover.scss
- ODS ICONS: /public/assets/img/ods
- NEW ICONS: public/assets/img/icons

### Comments
I created a JSON inside the HTML with a list of ODS, to serve it directly from the server (L: 264 of /Resources/templates/responsive/impact_discover/index.php)
I mocked up AJAX Calls at URL https://goteo-seven.vercel.app/ods.html?ods=
You can change this param on line 127 of /public/assets/js/impact_discover/impact_discover.js
Tested on Chrome & Safari

#### DESKTOP

1. Check if all elements fits like the design
    1. positions
    2. project img tinted
2. Click on a footprint. 
    1. Check if projects title has changed
    2. Check if ODS icons of the footprint are displayed
    3. Check if ODS select has filtered the ODS of the footprint
    4. Check if projects are reloaded and filtered by ODS (mocked by URL: https://goteo-seven.vercel.app/ods.html?ods=[ods])
3. Select an ODS
    1. Check if projects title has changed
    2. Check if ODS icons are hidden
    3. Check if footprints related are highlighted
    4. Check if projects are reloaded and filtered by ODS (mocked by URL: https://goteo-seven.vercel.app/ods.html?ods=[ods])
4. Remove an ODS via icons
    1. Check if ODS icon are removed
    2. Check if projects are reloaded and filtered by ODS (mocked by URL: https://goteo-seven.vercel.app/ods.html?ods=[ods])
5. Check if infinite scroll is working

#### MOBILE

1. Check if all elements fits like the design
    1. positions
    2. project img tinted
2. Open filters
    1. Check if footprints and ODS list are displayed
    2. Check close filter button
3. Open filters and click on a footprint. 
    1. Check if projects title has changed
    2. Check if ODS icons of the footprint are displayed
    3. Check if filters are hidden again
    4. Open filters and check if ODS list has filtered the ODS of the footprint
    5. Check if projects are reloaded and filtered by ODS (mocked by URL: https://goteo-seven.vercel.app/ods.html?ods=[ods])
4. Open filters and select some ODS
    1. Check if "borrar todos" is working
    2. Click on Aplicar filtros
        1. Check if filters are hidden again
        2. Check if projects title has changed
        3. Check if ODS icons filtered are displayed
        4. Check if projects are reloaded and filtered by ODS (mocked by URL: https://goteo-seven.vercel.app/ods.html?ods=[ods])
5. Check if infinite scroll is working

### CODE REVIEW

File: public/assets/sass/layouts/_impact_discover.scss [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+    
+                    &[data-footprint="all"] {
+                        background-color:#91B3B8;
+                        padding-left:0;
+                        padding-right:0;
+                        text-align:center;
+                    }
+    
+                    &[data-footprint="eco"] {
+                        background:url('../img/home/footprint-bg-tab-eco.png');
```
Comment (javicarrillo) [inline public/assets/sass/layouts/_impact_discover.scss:66]: The image is not in the branch.

File: public/assets/sass/layouts/_impact_discover.scss [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+                        background-size: cover;
+                    }
+    
+                    &[data-footprint="soc"] {
+                        background:url('../img/home/footprint-bg-tab-soc.png');
```
Comment (javicarrillo) [inline public/assets/sass/layouts/_impact_discover.scss:71]: The image is not in the branch.

File: public/assets/sass/layouts/_impact_discover.scss [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+                        background-size: cover;
+                    }
+    
+                    &[data-footprint="dem"] {
+                        background:url('../img/home/footprint-bg-tab-dem.png');
```
Comment (javicarrillo) [inline public/assets/sass/layouts/_impact_discover.scss:76]: The image is not in the branch.

**Review by javicarrillo — APPROVED** [review]

**Comment by davidbeig** [comment]
This PR is closed because it is overseeded by #200

**Pull request closed without merging**